src/api/commands/iteminformation/ItemInformationAddCommand.ts

```diff
@@ -41,6 +42,16 @@ export class ItemInformationAddCommand extends BaseCommand implements RpcCommand
      */
     @validate()
     public async execute( @request(RpcRequest) data: RpcRequest): Promise<ItemInformation> {
+        if (data.params.length < 5) {
+            this.log.error('Not enough args.');
+            throw new MessageException('Not enough args.');
+        } else if (typeof data.params[0] !== 'number') {
+            this.log.error('Listing template id must be numeric.');
+            throw new MessageException('Listing template id must be numeric.');
+        } else if (typeof data.params[4] !== 'number') {
+            this.log.error('Category id must be numeric.');
+            throw new MessageException('Category id must be numeric.');
+        }
         return this.itemInformationService.create({
```

Comment:
implement validate from RpcCommandInterface and move these new checks there

Reply:
also, use the model name 'ListingItemTemplate' instead of Listing template when referring to the model, and here when referring to the parameter, we should use the parameter name listingTemplateId when referring to that. Same with Category id, change to categoryId as that's how it's displayed in help.